include_physics/ideal_MHD.h

```diff
@@ -428,6 +428,29 @@ inline void compute_auxiliary_variables(REAL time, uint ix, uint iy, uint iz, gl
   set_field_component(ix, iy, iz, Field_p, u, p);
 }
 
+//--------------------------------------------------------------------------------------------------
+// Computation of analytical solution for all conserved variables
+//---------------------------------------------------------------------------------------------------
+
+inline void analytical_solution(uint ix, uint iy, uint iz, global REAL *u, REAL time) {
+
+  REAL rho = (REAL)(1);
```

Comment:
Minor/pedantic: The indentation does not seem to be uniform (using the github interface).